README.md

```diff
@@ -183,9 +183,9 @@ Binaries will be published once the codebase reaches a stable condition.
 | [yangrq1018](https://github.com/yangrq1018) | 4 |
 | [woshidama323](https://github.com/woshidama323) | 3 |
 | [crackcomm](https://github.com/crackcomm) | 3 |
-| [mshogin](https://github.com/mshogin) | 2 |
-| [herenow](https://github.com/herenow) | 2 |
-| [tk42](https://github.com/tk42) | 2 |
+| [azhang](https://github.com/azhang) | 2 |
```

Comment:
Can revert these changes since they're duplicates, then won't have to worry about it anymore once #1885 is merged